src/components/Orders.js

```diff
@@ -26,4 +27,9 @@ function Orders(props) {
     </div>);
 }
 
-export default Orders;
+const mapStateToProps = function (state) {
+  return {
+    Orders: state.Orders
```

Comment:
Go back and review the videos and lesson and address book to remember the pattern.